aws/table_aws_cloudtrail_trail_event_data_store.go

```diff
@@ -0,0 +1,184 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudtrailEventDataStore(_ context.Context) *plugin.Table {
```

Comment:
```suggestion
func tableAwsCloudTrailEventDataStore(_ context.Context) *plugin.Table {
```

The service name is `CloudTrail`, so `Trail` should be capitalized in the function name.